tests/test_biophysical/test_animal/test_cow_calf/test_beef_herd.py

```
@@ -0,0 +1,775 @@
+    assert (
+        reporter_spy.call_count == 4
+    ), f"Expected 4 reporter calls (one per beef animal), got {reporter_spy.call_count}"
+    called_animals = [c.args[0] for c in reporter_spy.call_args_list]
+    assert cow in called_animals
+    assert heifer in called_animals
+    assert calf in called_animals
+    assert bull in called_animals
+
+
+@pytest.mark.unit
+def test_initialize_beef_cow_calf_herd_non_dict_config_returns_empty(mocker: MockerFixture) -> None:
```

Ruff (0.15.20)

[warning] Unused function argument: `mocker`

(ARG001)